go.mod

```diff
@@ -223,3 +223,5 @@ require (
 
 // replace github.com/minio/console => ../console
 replace github.com/minio/console => github.com/minio/console v0.19.2-0.20220722194038-f26786c904d0
+
+replace github.com/minio/selfupdate => ../selfupdate
```

Comment:
Please of this make sure to point this `go.mod` to your `fork` @vadmeste - or use `go get github.com/minio/selfupdate@master` until its tagged.